pages/1.13/gui/secrets/index.md

```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
 ---
 
 You can manage secrets and certificates from the Secrets page.
-![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 
+![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 Figure 1 - Secrets page
+
+
+For complete details on creating and managing Secrets, see the [Secrets](/1.13/security/ent/secrets) documentation.
+
+<p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately 1 MB, subtracting approximately 1 KB for the secret store metadata.</p>
+
+# Creating key-value pair secrets 
+
+1. Log in to the DC/OS UI as a user with the `dcos:superuser` permission.
+
+1. Open the **Secrets** tab.
+
+1. Click the **+** icon in the top right.
+
+    ![New Secret](/1.13/img/new-secret.png)
+
+    Figure 2. New Secret icon
+
+1. In the **ID** box, provide the name of your secret and its path, if any.
+
+1. Type or paste the secret into the **Value** box.
+
+    ![Secret ID/Value Fields](/1.13/img/create-secret.png)
```

Comment:
Would you mind updating the screenshot please?

![Screenshot 2019-08-06 at 12 56 34](https://user-images.githubusercontent.com/186223/62534527-b167bd00-b849-11e9-8a63-bf4721b6f1dd.png)

Reply:
This is the old version of this modal dialogue. I have updated the image and it is listed here in this PR.